components/infobox/wikis/trackmania/infobox_league_custom.lua

```diff
@@ -116,6 +122,10 @@ function CustomLeague:defineCustomPageVariables(args)
 	Variables.varDefine('date', self.data.endDate)
 	Variables.varDefine('sdate', self.data.startDate)
 	Variables.varDefine('edate', self.data.endDate)
+
+	Variables.varDefine('circuit', self.data.circuit)
+	Variables.varDefine('circuittier', self.data.circuittier)
+	Variables.varDefine('circuitabbr', self.data.circuitabbr)
```

Comment:
where are those used? // are they needed?

Reply:
They are used in pages where we need to define the circuit. See https://liquipedia.net/trackmania/BIG_LAN_Race for an example

Reply:
to me it looks like they are not used at all

https://liquipedia.net/trackmania/index.php?title=Special:Search&profile=all&search=insource%3A%22circuitabbr%22&fulltext=1

currently there are no consumers for these wiki variables

Reply:
Removed.